1.6/Dockerfile.rhel7

```diff
@@ -46,9 +45,11 @@ RUN yum install -y tar zip unzip java-1.7.0-openjdk jenkins-1.609.1 && \
     mv oc /usr/local/bin && \
     rm oadm && \
     rm openshift && \
-    chmod -R og+rw /opt/openshift
+    mkdir -p /opt/openshift && chmod -R og+rw /opt/openshift && \
+    mkdir -p /usr/share/jenkins && chmod -R og+rw /usr/share/jenkins
 
-VOLUME ["/var/lib/jenkins"]
+COPY ./contrib/openshift /opt/openshift
+COPY ./contrib/jenkins /usr/local/bin
```

Comment:
@bparees @rhcarvalho I think this problem is not very related to this PR ;-) Can we move it to issues and continue discussing it there :-) I really want to have plugin installation infrastructure in place to move on the Jenkins R&D ;-)